src/app/Modules/auth/pages/forgot-password/forgot-password.component.md

```diff
@@ -0,0 +1,17 @@
+# Forgot Password Component
+
+Forgot password is for resetting the user password using their Email ID / Username.
+
+![Default Sidebar](images/forgot-password.png)
```

Comment:
Re the image - it's only necessary to show the dialog box - the background is not required and only makes the file huge and the picture harder to read. Just crop the dialog